### ISSUE TITLE
Problems running any sample test cases or tests

### ISSUE THREAD

**Issue by roboserg** [lgsvl/PythonAPI#52]
Every script under `\examples\SampleTestCases` throws an error `NameError: name 'env' is not defined`

Is it supposed to be like that without leaving out the two following likes?
```
from environs import Env
env = Env()
```

Secondly, even if I add above two lines, the map loads, I see the car spawning and then the simulation abruptly dies with 

```
Status: Error
Bridge socket was unexpectedly disconnected
```

**Comment by hadiTab** [comment]
@roboserg you are right. Those two lines are missing from those examples. Thanks for pointing it out!

As for the bridge connection error, that would occur if there is no bridge running. These examples are all trying to make a bridge connection to cyber bridge in the Apollo repo. If you do not have that running (or the address is different) you will receive a fatal bridge error.
